src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+import duke.exceptions.DukeException;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ *  Duke class that contains run and main method to run the program.
+ *
+ * @author Chua Sue-Ann
+ */
+public class Duke {
+
+    private duke.Storage storage;
+    private duke.TaskList taskList;
```

Comment:
Perhaps you want to consider importing duke.Storage and duke.TaskList classes above so you can simply refer to the class names? 🤔